src/libs/actions/connections/NetSuiteCommands.ts

```diff
@@ -94,6 +94,92 @@ function updateNetSuiteSubsidiary(policyID: string, newSubsidiary: SubsidiaryPar
     API.write(WRITE_COMMANDS.UPDATE_NETSUITE_SUBSIDIARY, params, onyxData);
 }
 
+function updateNetSuiteSyncTaxConfiguration(policyID: string, isSyncTaxEnabled: boolean) {
+    const onyxData: OnyxData = {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    connections: {
+                        netsuite: {
+                            options: {
+                                config: {
+                                    syncOptions: {
+                                        syncTax: isSyncTaxEnabled,
+                                    },
+                                    // pendingFields: {
```

Comment:
This is throwing type errors due to nesting. Will fix this in upcoming PRs.

Reply:
Sounds good, but then can we make the comment more descriptive than simply "Fixing in the future PR"?